static-site/src/components/ListResources/IndividualResource.jsx

```diff
@@ -0,0 +1,138 @@
+  font-size: ${resourceFontSize}px;
+  font-family: monospace;
+  white-space: pre; /* don't collapse back-to-back spaces */
+  color: ${(props) => props.hovered ? '#31586c' : '#5097BA'} !important;
+  text-decoration: none !important;
+`;
+
+function Name({displayName, hovered, href, topOfColumn}) {
+  return (
+    <ResourceLink href={href} target="_blank" rel="noreferrer" hovered={hovered}>
+      {'• '}{(hovered||topOfColumn) ? displayName.hovered : displayName.default}
+    </ResourceLink>
+  )
+}
+
+const Container = styled.div`
+  padding: 3px;
+  overflow: hidden;
+  color: #4F4B50;
+`;
+
+const FlexRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  gap: ${gapSize}px;
+  align-items: center;
+`;
+
+export function TooltipWrapper({description, children}) {
+  return (
+    <div
+      data-tooltip-id="listResourcesTooltip"
+      data-tooltip-html={description}
+      data-tooltip-place="top">
+      {children}
+    </div>
+  )
+} 
+
+export function IconContainer({Icon, text, handleClick=undefined, color="#aaa"}) {
+  const iconProps = {size: "1.2em", color};
+  const hasOnClick = typeof handleClick === 'function';
+  return (
+    <div
+      style={{display: 'flex', color, alignItems: 'center', gap: '3px', cursor: hasOnClick?'pointer':'auto'}}
+      onClick={hasOnClick ? handleClick : ()=>{}}
+    >
+      <Icon {...iconProps}/>
+      {text}
+    </div>
+  )
+}
+
+
+/**
+ * 
+ * @param {*} param0 
+ * @returns 
+ */
+export const IndividualResource = ({data, isMobile}) => {
+  const ref = useRef(null);
+  const [topOfColumn, setTopOfColumn] = useState(false);
+  useEffect(() => {
+    /* The column CSS is great but doesn't allow us to know if an element is at
+    the top of its column, so we resort to JS */
+    if (ref.current.offsetTop===ref.current.parentNode.offsetTop) {
+      setTopOfColumn(true);
+    }
+  }, []);
+
+  return (
+    <Container innerRef={ref}>
+
+      <FlexRow>
+
+        <ResourceLinkWrapper data={data}>          
+          <Name displayName={data.displayName} href={data.url} topOfColumn={topOfColumn}/>
+        </ResourceLinkWrapper>
+
+        {data.versioned && !isMobile && (
+          <TooltipWrapper description={`${data.nVersions} snapshots of this dataset available (click to see them)`}>
+            <IconContainer
+              Icon={MdHistory}
+              text={data.nVersions}
+            />
+          </TooltipWrapper>
+        )}
+
+      </FlexRow>
+
+    </Container>
+  )
+}
+
+
+/**
+ * Wrapper component which monitors for mouse-over events and injects a
+ * `hovered: boolean` prop into the child.
+ */
+export const ResourceLinkWrapper = ({children}) => {
+  const [hovered, setHovered] = useState(false);
+  const onMouseOver = useCallback(() => {
+    setHovered(true)
+  },[])
+  const onMouseOut = useCallback(() => setHovered(false), [setHovered]);
+  return (
+    <div>
+      <div onMouseOver={onMouseOver} onMouseOut={onMouseOut}>
```

Comment:
Yup, changed - I got in the habit of using `useCallback` because it makes it straightforward to memoise components or diff props. (If you don't then the function gets recreated each render and so it's always different.) But we don't use those performance techniques here so no need to use it!